packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -606,6 +607,29 @@ export abstract class BaseService extends Resource
     };
   }
 
+  /**
+   * This method is called to create a networkConfiguration.
+   */
+  // tslint:disable-next-line:max-line-length
+  protected configureAwsVpcNetworkingWithMultipleSecurityGroups(vpc: ec2.IVpc, assignPublicIp?: boolean, vpcSubnets?: ec2.SubnetSelection, securityGroups?: ec2.ISecurityGroup[]) {
```

Comment:
Whew! This is a mouthful  😅 -- thoughts about maybe taking out the "Multiple" so it's just `configureAwsVpcNetworkingWithSecurityGroups` instead?

Reply:
Done!